src/utils/runner.ts

```diff
@@ -0,0 +1,16 @@
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+export function runAtMidnight(callback: () => void) {
+	const midnight = <number><unknown>dayjs().tz('America/Sao_Paulo').endOf('day') + 1;
+	const msUntilMidnight = midnight - <number><unknown>dayjs();
```

Comment:
Took me a while to understand this line LOL

We can use this trick to convert it to number:

```suggestion
	const msUntilMidnight = midnight - +dayjs();
```

Reply:
Nice catch, will edit soon.